hash_practice/exercises.py

```diff
@@ -2,18 +2,50 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(nklog(k)), where k is the length of the string 
+        Space Complexity: O(log(n))

```

Comment:
This is incorrect. We still have to add a key-value pair to `result` for every element of `strings`, which means there will be O(n) key-value pairs added, giving us a space complexity of O(n).